external-docs/docs/guides/configuration/qube-config/columns/dimensions.md

```diff
@@ -0,0 +1,337 @@
+
+```json
+{ ...
+   "columns": {
+      "Year": {
+         "type": "dimension"
+      },
+      "Region": {
+         "from_template": "statistical-geography"
+      }
+   }
+}
+```
+
+## Inheritance
+
+To reuse or extend an existing dimension, the `from_existing` field can be configured to link to a URI where the
+dimension to be reused or extended is defined.
+
+To reuse a parent dimension without making any changes to it, set the `from_existing` field to the URI defining the
+dimension to be reused:
+
+```json
+{ ...
+   "columns": {
+      "Year": {
+         "type": "dimension",
+         "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod"
+      }
+   }
+}
+```
+
+To extend a parent dimension and create a new dimension from it, set the `from_existing` field to the URI defining the
+dimension to be reused, and set the `label` field to indicate that this is a new child dimension of
+`http://purl.org/linked-data/sdmx/2009/dimension#refArea`:
+
+```json
+{ ...
+   "columns": {
+      "Region": {
+         "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refArea",
+         "label": "Geographic region"
+      }
+   }
+}
+```
+
+## Advanced configuration
+
+### Cell URI templates
+
+!!! Warning
+    The use of the `cell_uri_template` field is considered an advanced configuration option, and therefore care must be taken to ensure that the values generated are valid.
+
+```json
+{ ...
+   "columns": {
+      "Song": {
+         "from_existing": "http://example.org/dimension/eurovision-songs",
+         "cell_uri_template": "http://example.org/code-lists/eurovision-songs/{+song}"
+      },
+      "Language": {
+         "code_list": "false",
+         "cell_uri_template": "http://example.org/eurovision-languages/{+language}"
+      }
+   }
+}
+```
+
+The `Song` and `Language` columns have both been configured with a `cell_uri_template` field. It is important to note
+that this field should only be used where the concept scheme is defined externally at an existing URI, or there is no
+concept scheme, but you want to point to an existing resource to provide additional context about the dimension's value.
+
+If `cell_uri_template` is specified:
+
+**Either**
+:     `from_existing` must also be defined, in which case `cell_uri_template` should refer to the concepts in the existing
+      dimension's code list;
+**Or**
+:     `code_list` must be set as `false`, in which case `cell_uri_template` should refer to URIs which are existing RDF
+      resources.
+
+The format of the `cell_uri_template` value **must** follow [RFC6570](https://www.rfc-editor.org/rfc/rfc6570) guidance
+for URI Templates. In the case of any doubt, follow the pattern in the examples shown above (e.g.
+`http://example.org/some-uri/{+column_name}`), as this will ensure csvcubed safely
+[transforms the column header](../../../uris.md#csv-column-name-safe-transformation) to the CSV-W format.
```

Comment:
I don't think the csv-column-name-safe-transformation link is in the right context here. It's more that we need to let the user know that the variables they use inside the template are the values after the CSV column title has gone through the csv-column-name-safe-transformation.

Reply:
Does this make more sense?